packages/client/src/v2-events/features/events/AdvancedSearch/SearchResult.tsx

```diff
@@ -173,14 +181,59 @@ const ExtendedEventStatuses = {
   DRAFT: 'DRAFT'
 } as const
 
+function ActionComponent({
+  event,
+  actionType
+}: {
+  event: EventIndex
+  actionType: WorkqueueActionsWithDefault
+}) {
+  const { slug } = useTypedParams(ROUTES.V2.WORKQUEUES.WORKQUEUE)
+
+  const { config: configs } = useAction(event)
+  const actionMenuItems = useActionMenuItems(event)
+
+  const intl = useIntl()
+
+  let config: ActionConfig | undefined
+  if (actionType === 'DEFAULT') {
+    config = actionMenuItems.find(
+      ({ type }) =>
+        !(
+          [
+            ActionType.ASSIGN,
+            ActionType.UNASSIGN,
+            ActionType.READ
+          ] as readonly string[]
```

Comment:
This same array appears in many place already I think, e.g. here: https://github.com/opencrvs/opencrvs-core/blob/develop/packages/commons/src/events/scopes.ts#L122-L127

Maybe its time to save it to constants somewhere? For example something like:

```
const META_ACTIONS: ActionType[] = [
  ActionType.ASSIGN,
  ActionType.UNASSIGN,
  ActionType.READ
]

function isMetaAction(actionType: ActionType) {
  return META_ACTIONS.includes(actionType)
}
```

With this we don't need to cast anymore:

```
  const config =
    actionType === 'DEFAULT'
      ? actionMenuItems.find(({ type }) => !isMetaAction(type))
      : configs[actionType]
```